sig-node/sig-node-contributor-ladder.md

```diff
@@ -114,3 +114,29 @@ There are different ways to earn trust:
 
 **Be present**
 Participate in SIG Node meetings by speaking about KEPs or improvements driven, or find some other way to prove the identity behind GitHub handle.
+
+## Emeritus Approvers
+
+The [emeritus_approvers](https://www.kubernetes.dev/docs/guide/owners/#emeritus) section is used
+to list approvers who may no longer have time to spend on the project regularly. Keeping the list of
+_active_ approvers up to date helps contributors find the approver for their work easier.
```

Comment:
WDYT of 
`approvers up to date helps contributors find an approver for their work`